src/elementTree.js

```diff
@@ -283,9 +280,9 @@ function processToken(token, source) {
         token.type = 'Punctuator';
         token.sourceCode = token.value = '>';
     } else if (type === tt.jsxName) {
-        token.type = 'JSXIdentifier';
+        token.type = 'Identifier';
     } else if (type === tt.jsxText) {
-        token.type = 'JSXText';
+        token.type = 'String';
```

Comment:
Yep, I also see Literal, but literal is based on `JSXText` token. Which, seems like, we have to support. Because `JSXText` is not a string for reasons I provided above.